src/core/build_target.go

```diff
@@ -1218,6 +1230,14 @@ func (target *BuildTarget) AddTestTool(tool BuildInput) {
 	}
 }
 
+// AddDebugTool adds a new debug tool to the target.
+func (target *BuildTarget) AddDebugTool(tool BuildInput) {
```

Comment:
We might need to think about whether this stuff should contribute to the build hash.

Reply:
Ya, let's catch up on this